src/controllers/tracker.controller.js

```diff
@@ -0,0 +1,13 @@
+import commandMap from "./commands/commandMap.js";
+
+export function trackerController(bot) {
+  console.log("commands", commandMap);
+  Object.entries(commandMap).forEach((entry) => {
+    console.log(entry[0], entry[1]);
+    console.log(typeof entry[0]);
```

Comment:
Should we move to better logging library? 
https://www.npmjs.com/package/winston

Reply:
sure. but out of scope for this PR

Reply:
see #26